doc/ref_model/chapters/appendix-a.md

```diff
@@ -1,28 +1,25 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines For VNF Vendors
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 Goals](#A.1)
-* [A.2 Intro and Terminology](#A.2)
-* [A.3 VNF Maturity Levels](#A.3)
-* [A.4 Links](#A.3)
+* [A.1 Guidelines for VNF Vendors](#A.1)
+* [A.2 Guidelines for CNF Vendors](#A.2)
+* [A.3 VNF Transition Plan](#A.3)
+* [A.4 CNF Transition Phases](#A.4)
```

Comment:
@rabi-abdel should A.4 be consistent with A.3 -- plans versus phases

Reply:
@pgoyal01 yes. fixed.